Quaver.Shared/Screens/Edit/UI/Playfield/EditorHitObject.cs

```diff
@@ -145,7 +145,7 @@ public void SetPosition()
         /// </summary>
         public Texture2D GetHitObjectTexture()
         {
-            var index = SkinMode.ColorObjectsBySnapDistance ? HitObjectManager.GetBeatSnap(Info, Info.GetTimingPoint(Map.TimingPoints)) : 0;
+            var index = SkinMode.ColorObjectsBySnapDistance ? HitObjectManager.GetBeatSnap(Info, Info.GetTimingPoint(Map.TimingPoints), true) : 0;
```

Comment:
Could we avoid changing GetBeatSnap() to have a special case for editor objects by just using BeatSnapToSnapIndex(Info.Color) here?